packages/@aws-cdk/aws-kinesis/lib/stream.ts

```diff
@@ -210,7 +213,8 @@ export interface StreamProps {
    *
    * The 'encryption' property must be set to "Kms".
    *
-   * @default - If encryption is set to "KMS" and this property is undefined, a
+   * @default - Kinesis Data Streams master key ('/alias/aws/kinesis')
+   * If encryption is set to StreamEncryption.KMS and this property is undefined, a
```

Comment:
Same (indent the continuation of the documentation, separate the two sentences with some punctation - otherwise it will render as "Kinesis Data Streams master key ('/alias/aws/kinesis') If encryption is set to ..." in the docs, which looks weird).

Reply:
done